influxdb/rootfs/etc/services.d/influxdb/run

```diff
@@ -3,6 +3,16 @@
 # Community Hass.io Add-ons: InfluxDB
 # Runs the InfluxDB Server
 # ==============================================================================
+declare name
+declare value
+
+for envvar in $(bashio::config 'envvars|keys'); do
+    name=$(bashio::config "envvars[${envvar}].name")
+    value=$(bashio::config "envvars[${envvar}].value")
+    bashio::log.debug "Setting Env Variable ${name} to ${value}"
+    export "${name}"="${value}"
```

Comment:
We can do this is a single string, especially with the tighter regular expression.

```suggestion
    export "${name}=${value}"
```

Reply:
Think this was just more a habit thing 😉

Reply:
🤷‍♂ Yeah maybe.